### DISCUSSION TITLE
link-like path for button

### DISCUSSION THREAD

**Discussion by azim-b** [h2oai/wave#1318] [Q&A] [answered]
Can `ui.button()` take you to a `path`? For example, `ui.lin()`'s `path` argument.

Context: I'd like to open the email client with pre-populated info for support.

For example, I could use a `ui.link()` to make a `mailto:` path.

```
support_email_path = f"""
mailto:support@h2o.ai?
cc=azim.baghadiya@&
subject=Error Support&
body= Sample Wave App Error. Please add more details below.%0A%0A
"""
support_email_path = support_email_path.replace(" ", "%20")
ui.link(label='Support', path=support_email_path)
 ```

Please note that the button is **Primary** and also has a Help-like **icon**, due to which we're unable to use a `ui.link()` with the `button` param set.

TIA.

**Comment by mturoci** [comment, marked as answer]
This seems like a pretty frequent request, tracked in https://github.com/h2oai/wave/issues/1329
